pkg/convert/xliff12_test.go

```diff
@@ -101,6 +103,33 @@ func Test_FromXliff12(t *testing.T) {
 			data:     randXliff12(nonOriginalTranslation),
 			expected: nonOriginalTranslation,
 		},
+		{
+			name:                "Message with special chars {}",
+			containsSpecialChar: true,
+			data: randXliff12(
+				&model.Translation{
+					Language: language.English,
+					Original: false,
+					Messages: []model.Message{
+						{
+							ID:      "order canceled",
+							Message: "{Order #{Id} has been canceled for {ClientName} | \\}",
+						},
+					},
+				},
+			),
+			expected: &model.Translation{
+				Original: false,
+				Language: language.English,
+				Messages: []model.Message{
+					{
+						ID:      "order canceled",
+						Message: `{Order #\{Id\} has been canceled for \{ClientName\} \| \\}`,
```

Comment:
Shouldn't this be -> `{Order #\{Id\} has been canceled for \{ClientName\} \| \\\\}` ?

Reply:
The expected Message is correct. since string "\\" is equal to single `\` in backticks. 
I will also wrap the input message in backticks for readability.